script/Arcueid/Day 13/13_00_ARC13_1.txt

```diff
@@ -492,52 +507,55 @@ Why would the old man tell me something like that...?
 [sha:be44d6651070e71407233cdd7ad7b72755eecd98]{
 -- Page 34, Offset 31766.
 -- 秋葉はすべて……七年前の事も、あのシキの事も、全部知っているのか？」
-Akiha... you knew about everything, including what happened seven years ago and SHIKI?"
+Akiha, you knew everything? Even what happened seven years ago, and about SHIKI?"
 }
 [sha:7b5518fec7046de3940bd166a72499335dfa70cd]{
 -- Page 34, Offset 31767.
 -- 「……はい。シキの事は、兄さんには思い出してほしくなかった。出来る事なら、ずっと忘れたままでいてほしかった」
-"...Yes. I didn't want you to remember SHIKI at all. In fact, I wanted you to remain ignorant forever."
+"...Yes. I didn't want you to remember anything about SHIKI.
+ I'd hoped for you to remain blissfully ignorant forever, if at all possible."
 }
 [sha:7811de415accee1d597f320ea8f3f6e9eb4fe3c4]{
 -- Page 35, Offset 31768. Mods: @n.
 -- 「……けど、
-"...However,
+"...Alas,
 }
 [sha:a943d806cac696885695f3176a068553645f0e82]{
 -- Page 35, Offset 31769. Glued.
 -- それもおしまいですね。
- even that has failed.
+ it seems that I have failed.
 }
 [sha:3516fd46a3f57a43a7f9eb18d94ea1bdc894fa17]{
 -- Page 35, Offset 31770.
 -- はじめから……隠し通す事なんて、無理な話だったんです」
-Hiding that fact... was an impossible task from the very beginning."
+Keeping his existence hidden... was an impossible task to begin with."
 }
 [sha:4760ef83a0502b88b122ac85d6b06f7ba0138b84]{
 -- Page 35, Offset 31771.
 -- そう言って、どこか自嘲ぎみに笑って。
-Saying that, she chuckles to herself out of self-deprecation of some sort.
+// そう言って not really adding much here
+She chuckles derisively at herself.
 }
 [sha:40f208daa9efc5523a569af968c0f291932da0e2]{
 -- Page 35, Offset 31772.
 -- 秋葉はまっすぐに俺の目を見て話を始めた。
-She looks straight into my eyes and begins to talk.
+Turning to look me straight in the eye, Akiha begins to talk.
 }
 [sha:4c4b5d660365d6cc556972e8c56a088350db8abf]{
 -- Page 36, Offset 31773.
 -- 「兄さん。遠野の家が特別な血族だという事はあの人たちに聞いたでしょう？　信じられないでしょうけど、遠野の血は人間以外の血が混ざっているんです。
-"Nii-san. You've heard from those people that the Tohno family is a special bloodline, correct? It's hard to believe, but the Tohno blood is mixed with the blood of something non-human.
+"Nii-san. You've heard from those people that the Tohno family is a special bloodline, correct?
+ This may be hard to believe, but the Tohno blood is mixed with that of something non-human.
 }
 [sha:8b6883cb106e113ddf048a66148ba02cc39e6b1e]{
 -- Page 36, Offset 31774.
 -- ……少なくとも、私は幼い頃からお父様にそう教えられて育ってきました」
-...At least, that's what father taught me when I was young and raised me as such."
+...At least, that's what father drilled into me the entire time I was growing up."
```

Comment:
I leant away from it because it felt slightly in conflict with the next line where she says she _didn't_ believe it (at first), and wanted to preserve the verb as being told/instructed over the course of growing up so that the belief can be more in flux.

Reply:
then alternatively "drilled into me for my entire childhood"

Reply:
Nitpicking but "...into me ever since I was a child" just so that we don't lose the ~から? I assume that he kept harping on about until he died.

Reply:
if you think that nuance is essential then that's fine. I just think that it doesn't read as well.

Reply:
at this point I've stared at it so long I'm no longer sure which formulation feels better. I'll go with yours.